packages/ember-runtime/lib/mixins/target_action_support.js

```diff
@@ -159,5 +165,23 @@ function getTarget(instance) {
     }
   }
 
+  registerHandler(function(message, options, next) {
```

Comment:
I think we should remove this line and the line below that does `target = get(instance, 'targetObject')` (so that we don't trigger the deprecation).

Things should fall through now that we have the getter/setter above.